tensorflow/contrib/libsvm/BUILD

```diff
@@ -0,0 +1,102 @@
+package(
```

Comment:
Adding a new `BUILD` file will require complementary changes in `tensorflow/contrib/cmake/tf_core_kernels.cmake` and `tensorflow/contrib/cmake/tf_python.cmake`.

Reply:
Thanks @mrry. The PR has been updated with cmake file added. I don't have a Windows dev machine setup yet but will try to see if I could fix any issues if encountered. May get a Windows dev machine later.